src/main/api/ledger.ts

```diff
@@ -0,0 +1,10 @@
+import AppBtc from '@ledgerhq/hw-app-btc'
+import TransportNodeHid from '@ledgerhq/hw-transport-node-hid'
+
+export const getBTCAddress = async () => {
+  const transport = await TransportNodeHid.open('')
+  const appBtc = new AppBtc(transport)
+  const info = await appBtc.getWalletPublicKey("44'/0'/0'/0/0")
+  await transport.close()
+  return info.bitcoinAddress
```

Comment:
In case of errors (e.g. no Ledger has been connected), handle errors by rejecting returned Promise.

```
Error occurred in handler for 'GET_BTC_ADDRESS': Error: 
[dev:electron]     at new TransportError (asgardex-electron/node_modules/@ledgerhq/hw-transport-node-hid/node_modules/@ledgerhq/errors/dist/index.cjs.js:219:18)
[dev:electron]     at asgardex-electron/node_modules/@ledgerhq/hw-transport-node-hid/src/TransportNodeHid.js:121:26
[dev:electron]     at Object.exports.getBTCAddress (asgardex-electron/src/main/api/ledger.ts:5:21)
[dev:electron]     at electron/js2c/browser_init.js:189:542 {
[dev:electron]   message: 'NoDevice',
[dev:electron]   id: 'NoDevice'
[dev:electron] }
```

Reply:
added error handler